homeassistant/components/unifi/device_tracker.py

```diff
@@ -237,6 +241,9 @@ def device_state_attributes(self):
 
         attributes["is_wired"] = self.is_wired
 
+        if "hostname" in attributes:
```

Comment:
This almost worked with no changes, but `device_tracker` has standardized on `host_name` and unifi uses `hostname`